packages/design-system/token-transformer/README.md

```diff
@@ -0,0 +1,3 @@
+# @eternagame/design-system-token-transformer
```

Comment:
Yeah, this was a point of confusion for me with the generator; what is the package name vs. the package's position within the file structure. Would like to talk about what your intention was originally.

Reply:
My decision to have the "parent directory" being included by default when generating packages was semi-arbitrary and based on Nx's behavior (or at least how Nx previously behaved - I'm not sure if it holds true with Nx 15). It seemed like prefixing it could be a common enough use case to warrant it being the default behavior, but maybe that's not correct. I'm open for discussion if that default should change and what our convention in this repo should be